app/assets/stylesheets/style.scss

```diff
@@ -7,6 +7,22 @@
   font-size: rem-calc(14);
 }
 
-.advanced-search li {
-  list-style: none;
+.has-form {
+ padding: 0px 0px 0px 0px;
```

Comment:
Line should be indented 2 spaces, but was indented 1 spaces<br>Shorthand form for property `padding` should be written more concisely as `0px` instead of `0px 0px 0px 0px`<br>`0px` should be written without units as `0`